src/index.tsx

```diff
@@ -1,6 +1,8 @@
 import * as React from "react";
 import { render } from "react-dom";
 import App from "./components/App";
+import "bootstrap/dist/css/bootstrap.css";
```

Comment:
nit: I think you can just do `import 'bootstrap'` since you npm installed it: https://getbootstrap.com/docs/4.0/getting-started/webpack/#importing-javascript

Reply:
^that didn't work right away, but I'm sure this will come up again so we can figure out why